daemons/gptp/common/ptp_message.cpp

```diff
@@ -1467,6 +1469,8 @@ void PTPMessagePathDelayRespFollowUp::processMessage(IEEE1588Port * port)
 		 remote_req_rx_timestamp.nanoseconds);
 
 	// Adjust turn-around time for peer to local clock rate difference
+	// TODO: Are these .998 and 1.002 specifically defined in the standard?
```

Comment:
The numbers I put here are somewhat arbitrary. I have in the past used hardware that occasionally returned bad timestamps. Since it seemed like a difficult task to examine a single timestamp and decide whether it was good, I completed the calculation, and if, for example, one of the timestamps was bad, the result would be way off and the calculation would be rejected. There is probably a better way to accomplish this, but it did it's job at the time. I don't really have an opinion as whether to keep the code. It probably can go.